File: 1-two-sum/1-two-sum.kt
class Solution {
    fun twoSum(nums: IntArray, target: Int): IntArray {
        var a = 0
        var b = 0
        for (i in nums.indices) {
            for (j in i + 1 until nums.size) {
                if (nums[i] + nums[j] == target) {
                    a = i
                    b = j
                    break
                }
            }
        }

        return intArrayOf(a, b)
    }
}

File: 1-two-sum/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let a = 0;
    let b = 0;
    for(let i = 0; i < nums.length ; i++){
      for(let j = i + 1; j < nums.length; j++){
          if (nums[i] + nums[j] === target) {
            a = i
            b = j
            break;
          }
      }
    } 
    return [a,b]
};